regression/linting.ts

```diff
@@ -0,0 +1,83 @@
+// import * as autorest from "autorest"
+import { exec } from "child_process"
+
+function genLintingCmd(readme: string): string {
+  const cwd = process.cwd()
+  const linterCmd = `npx autorest ${readme} --azure-validator=true --validation --message-format=json --use=${cwd}/src/typescript --use=${cwd}/src/dotnet/AutoRest`
+  return linterCmd
+} 
+
+export async function runLinter(readme: string) {
+  const linterCmd = genLintingCmd(readme)
+  try {
+    let linterErrors = await runCmd(linterCmd)
+    if (linterErrors.indexOf('{\n  "type": "') !== -1) {
+      linterErrors = cleanUpContent(linterErrors)
+      const errorJsonStr = "[" + linterErrors + "]"
+      const errorJson = JSON.parse(errorJsonStr).sort( (a,b) => {
+        let isLess = 0;
```

Comment:
What does isLess do？

Reply:
It's the sort function's return value, may be -1,0,1 each indicate  < , = , >